app/views/case_logs/_log_list.html.erb

```diff
@@ -1,29 +1,46 @@
-<table class="govuk-table">
-  <caption class="govuk-table__caption govuk-table__caption--m"><%= title %></caption>
-  <thead class="govuk-table__head">
-    <tr class="govuk-table__row">
-      <th scope="col" class="govuk-table__header">Log reference</th>
-      <th scope="col" class="govuk-table__header">Postcode</th>
-      <th scope="col" class="govuk-table__header">Tenant code</th>
-      <th scope="col" class="govuk-table__header"><%= date_title %></th>
-    </tr>
-  </thead>
-  <tbody class="govuk-table__body">
-    <% case_logs.map do |log| %>
-    <tr class="govuk-table__row">
-      <th scope="row" class="govuk-table__header">
-        <%= govuk_link_to log.id, case_log_path(log) %>
-      </th>
-      <td class="govuk-table__cell govuk-table__cell">
-        <%= log.property_postcode %>
-      </td>
-      <td class="govuk-table__cell govuk-table__cell">
-        <%= log.tenant_code %>
-      </td>
-      <td id="last-changed" class="govuk-table__cell">
-        <%= log.updated_at.to_formatted_s(:govuk_date) %>
-      </td>
-    </tr>
-    <% end %>
-  </tbody>
-</table>
+<figure class="app-figure">
+  <figcaption id="<%= title.dasherize %>" class="app-figure__caption">
+    <b><%= case_logs.count %></b> <%= title.downcase %>
```

Comment:
You probably want `case_logs.size` here rather than `.count` since you've already loaded them. `.count` will end up doing too SQL queries:

```
CaseLog Load (2.0ms)  SELECT "case_logs".* FROM "case_logs" WHERE ("case_logs"."owning_organisation_id" = $1 OR "case_logs"."managing_organisation_id" = $2) 

CaseLog Count (1.7ms)  SELECT COUNT(*) FROM "case_logs" WHERE ("case_logs"."owning_organisation_id" = $1 OR "case_logs"."managing_organisation_id" = $2)
```

Whereas `.size` here would just do the first.